### PR TITLE
Adjust grammar specification for Primary Constructors.

### PR DESCRIPTION
**Pull request by AlekseyTs**

### CODE REVIEW

**Review by 333fred — APPROVED** [review]

File: proposals/primary-constructors.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ class_body
   
 struct_declaration
   : attributes? struct_modifier* 'partial'? 'record'? 'struct' identifier type_parameter_list?
-    parameter_list? struct_interfaces? type_parameter_constraints_clause* struct_body ';'?
+    parameter_list? struct_interfaces? type_parameter_constraints_clause* struct_body
```
Comment (CyrusNajmabadi) [inline proposals/primary-constructors.md:67]: i thought we allowed trailing semicolon on thsee types in the language.  or is this only for error recovery?

**Review by CyrusNajmabadi — APPROVED** [review]

File: proposals/primary-constructors.md (hunk 1 above)
Comment (MadsTorgersen) [reply proposals/primary-constructors.md:67]: We do, but it's already built into the `class_body` and `struct_body` productions, so also having it here weirdly allows up to two trailing semicolons!

**Review by MadsTorgersen — APPROVED** [review]
This change seems right. The `class_body` and `struct_body` productions already allow trailing semicolon, so `class_declaration` and `struct_declaration` shouldn't also do so.

**Comment by CyrusNajmabadi** [comment]
ah!  that makes sense.  Thanks! :)

**Pull request merged**